src/environments/environment.ts

```diff
@@ -10,7 +10,8 @@ export const environment = {
   get_all_borrowers: '/borrowers',
   deleteBorrower: '/borrower/',
   createBorrower: '/borrower',
-  deleteAuthor: '/deleteAuthor/'
+  deleteAuthor: '/deleteAuthor/',
+  updateBorrower: '/borrower/'
```

Comment:
Do we really need to have separate identical endpoint strings in `environment`, when the backend distinguishes them by HTTP verb?

Reply:
I will make those changes